sdk/monitor/Microsoft.OpenTelemetry.Exporter.AzureMonitor/src/ApplicationInsightsRestClient.cs

```diff
@@ -10,8 +10,13 @@
 
 namespace Microsoft.OpenTelemetry.Exporter.AzureMonitor
 {
+    /// <summary>
+    /// This is the custom class. The other partial is the Swagger auto-generated class.
+    /// </summary>
     internal partial class ApplicationInsightsRestClient
     {
+        internal string ApiVersion { get; set; } = "2";
```

Comment:
Do we want this to be `readonly`?

Reply:
Yes and no...... 
This class is weird... half of it is ours and the other half is generated from the swagger. 
To change this to be readonly, it would need to be set from the constructor, which belongs to the swagger half.

The RestClient needs to be refactored because we're currently only using half of the generated class. I think that is out of scope of this minor change, but I would like to discuss what we want this class to look like